packages/core/src/domain/configuration/experimentalFeatures.spec.ts

```diff
@@ -37,3 +38,14 @@ describe('experimentalFeatures', () => {
     expect(isExperimentalFeatureEnabled('foo')).toBeTrue()
   })
 })
+
+describe('setSampledExperimentalFeatures', () => {
+  it('should sample experimental features', () => {
+    setSampledExperimentalFeatures({ foo: 100, bar: 0 })
+    updateExperimentalFeatures(['foo', 'bar', 'baz'])
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏what about allowing us to configure the sampling rate from the init configuration?
```ts
enableExperimentalFeatures: [{foo: 100}, {bar: 0}, 'baz']
```
it would:

- be closer to what other DD feature flags can do
- allow us to change the rate without updating SDK code

cc @BenoitZugmeyer

Reply:
Sounds nice. Lets do it

Reply:
Done! Let me know what you think

Reply:
Yes it looks nice! A follow-up could be to list enabled FF in telemetry events, so we can compare A/B data.